LayoutTests/platform/mac-wk2/fast/dom/navigator-detached-no-crash-expected.txt

```diff
@@ -32,6 +32,7 @@ navigator.setLoggedIn() is OK
 navigator.setLoggedOut() is OK
 navigator.share() is OK
 navigator.storage is OK
+navigator.userActivation is OK
```

Comment:
Nice test. Do we have WPT coverage for isActive/hasBeenActive values for detached frames?

Reply:
No. I'll add one.

Reply:
So, if I'm reading the [HTML spec correctly](https://html.spec.whatwg.org/#user-activation-data-model), it always assumes a window being there (i.e., this is undefined behavior). Our implementation also doesn't check if there is a window attached. 

We might need to file a bug on HTML.

Reply:
Yeah, seems worth filing a bug. For now, I think it's OK to write a test case even if the behavior is not yet specified. The best test case tells you that you have correct behavior, and also guards against accidental change. But a test case that only guards against accidental change is also valuable.

Reply:
(For WebKit test cases, anyway. For WPT, it's probably rude to check in a test case if you don't know what the correct behavior is. :P )

Reply:
Will add a test ... filed https://github.com/whatwg/html/issues/8378

Reply:
Test was added https://github.com/web-platform-tests/wpt/pull/36684 and included with this patch.